docker/kernel/httploader/httploader/falcoloader/loader.go

```diff
@@ -0,0 +1,148 @@
+package falcoloader
+
+import "io/ioutil"
+import "io"
+import "os"
+import "crypto/md5"
+import "encoding/hex"
+import "strings"
+import "net/http"
+import "golang.org/x/sys/unix"
+import "log"
+import "unsafe"
+import "compress/gzip"
+import "bytes"
+
+func GetKernelVersion() string {
+	path := "/proc/version"
```

Comment:
Using `/proc/sys/kernel/osrelease` avoid us the following (string split etc.) logic.

Reply:
Otherwise you can use `syscall.Uname` method ([docs](https://golang.org/src/syscall/zsyscall_linux_amd64.go?s=22199:22235#L928)).